.github/workflows/update-tempest-releases.yaml

```diff
@@ -0,0 +1,35 @@
+name: Update Tempest Releases
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '43 1 * * 1'
+
+jobs:
+  update-antelope:
+    uses: ./.github/workflows/update-snapcraft.yaml
+    with:
+      openstack-release: antelope
+  update-zed:
+    uses: ./.github/workflows/update-snapcraft.yaml
+    with:
+      openstack-release: zed
+  update-yoga:
+    uses: ./.github/workflows/update-snapcraft.yaml
+    with:
+      openstack-release: yoga
+  update-xena:
+    uses: ./.github/workflows/update-snapcraft.yaml
+    with:
+      openstack-release: xena
+  update-wallaby:
+    uses: ./.github/workflows/update-snapcraft.yaml
+    with:
+      openstack-release: wallaby
+  update-victoria:
+    uses: ./.github/workflows/update-snapcraft.yaml
+    with:
+      openstack-release: victoria
+  update-ussuri:
+    uses: ./.github/workflows/update-snapcraft.yaml
+    with:
+      openstack-release: ussuri
```

Comment:
Instead of creating one job per release, I'm think if we let `update-snapcraft.yaml` accept a list of openstack releases instead of a single one and use [jobs.<job_id>.strategy.matrix](https://docs.github.com/en/enterprise-cloud@latest/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstrategymatrix) to run the update for each release, we could achieve the same goal with a single job here:
```
jobs:
  update-releases:
    uses: ./.github/workflows/update-snapcraft.yaml
    with:
      openstack-releases: "['antelope', 'zed', 'yoga', 'xena', 'wallaby', 'victoria', 'ussuri']"
```
This way we avoid repeating similar codes, and also if we need to add a release in the future, we can simply extend the list

Reply:
hey, thanks for this! indeed it looks like a much better solution, i should've read the workflow docs better :sweat_smile:  

Implemented it and pushed an update. will put up a comment with links to my trial run in my fork. cheers.